data_steward/utils/pipeline_logging.py

```diff
@@ -16,14 +16,16 @@
 
 # Project imports
 import resources
+from app_identity import get_application_id
 
 DEFAULT_LOG_DIR = os.path.join(resources.base_path, 'logs')
 """Default location for log files"""
 DEFAULT_LOG_LEVEL = logging.INFO
 
 LOG_FORMAT = '%(asctime)s - %(levelname)s - %(name)s - %(message)s'
 LOG_DATEFMT = '%Y-%m-%d %H:%M:%S'
-FILENAME_FMT = '%Y%m%d-' + os.getenv('GOOGLE_CLOUD_PROJECT', 'UNDEFINED_GOOGLE_PROJECT')
+app_id = get_application_id()
+FILENAME_FMT = f'%Y%m%d-{app_id}'
```

Comment:
This would also work.
```
FILENAME_FMT = f'%Y%m%d-{get_application_id()}'
```

Reply:
I actually tried both, verified they both work, but in random googling I found more examples of this usage, so I assumed it was more "pythonic"